templates/store/user-details.html

```diff
@@ -0,0 +1,110 @@
+{% extends "_layout.html" %}
+
+{% block title %}{{ context.username }}{% endblock %}
+
+{% block content %}
+
+<div class="p-strip--light is-bordered is-shallow" data-js="sticky-header">
+  <header class="row">
+    <div class="p-media-object--large u-no-margin--bottom">
```

Comment:
To simplify this template lets remove the media-object here and leave it as just a H1 followed by p-inline-list.

For example:
``` html
<header class="row">
  <h1 itemprop="name" style="text-transform:capitalize">
    {{ context.username }}
  </h1>
  <ul class="p-inline-list">
    <li class="p-inline-list__item">
      <a href="#bundles">{{ context.bundles_count }} bundle{{ context.bundles_count|pluralize }}</a>
    </li>
    <li class="p-inline-list__item">
      <a href="#charms">{{ context.charms_count }} charm{{ context.charms_count|pluralize }}</a>
    </li>
  </ul>
</header>
```

Reply:
Done.